docs/tutorials/integrations/kserve-mm/blue-green.md

```diff
@@ -0,0 +1,238 @@
+---
+template: main.html
+---
+
+# Blue-Green Rollout of a ML Model
+
+This tutorial shows how Iter8 can be used to implement a blue-green rollout of ML models hosted in a KServe modelmesh serving environment. In a blue-green rollout, a percentage of inference requests are directed to a candidate version of the model. The remaining requests go to the primary, or initial, version of the model. Iter8 enables a blue-green rollout by automatically configuring the network to distribute inference requests.
+
+After a one time initialization step, the end user merely deploys candidate models, evaluates them and either promotes or deletes them. Optionally, the end user can modify the percentage of inference requests being sent to the candidate model. Iter8 automatically handles all underlying network configuration.
+
+![Blue-Green rollout](images/blue-green.png)
+
+In this tutorial, we use the Istio service mesh to distribute inference requests between different versions of a model.
+
+???+ "Before you begin"
+    1. Ensure that you have the [kubectl CLI](https://kubernetes.io/docs/reference/kubectl/).
+    2. Have access to a cluster running [KServe ModelMesh Serving](https://github.com/kserve/modelmesh-serving) and [Istio](https://istio.io). For example, you can create a modelmesh-serving [Quickstart](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md) environment and install a [demo version](https://istio.io/latest/docs/setup/getting-started/) of Istio. Ensure `istioctl` is in your path.
+
+## Install the Iter8 controller
+
+--8<-- "docs/tutorials/integrations/kserve-mm/installiter8controller.md"
+
+## Deploy a primary model
+
+Deploy the primary version of a model using an `InferenceService`:
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: wisdom-0
+  labels:
+    app.kubernetes.io/name: wisdom
+    app.kubernetes.io/version: v1
+    iter8.tools/watch: "true"
+  annotations:
+    serving.kserve.io/deploymentMode: ModelMesh
+    serving.kserve.io/secretKey: localMinIO
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      storageUri: s3://modelmesh-example-models/sklearn/mnist-svm.joblib
+EOF
+```
+
+??? note "About the primary `InferenceService`"
+    Naming the model with the suffix `-0` (and the candidate with the suffix `-1`) simplifies the rollout initialization. However, any names can be specified.
+    
+    The label `iter8.tools/watch: "true"` lets Iter8 know that it should pay attention to changes to this InferenceService.
```

Comment:
```suggestion
    The label `iter8.tools/watch: "true"` lets Iter8 know that it should pay attention to changes to this `InferenceService`.
```

Most `InferenceService` seem to have the code highlight. I'm not sure if this is the only one missing it.